lection3/task2.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+// 1) x2 - 6x + 9 = 0
+// 2) x2 - 4x - 5 = 0
+
+function calculate(a, b, c) {
+    let radical1, radical2, discriminant, answer;
```

Comment:
All this variables will not be changed, so i belive "const" should be used instead

Reply:
Not all. "answer" can be num or string - so I decide to leave it with let